### PR TITLE
Install Serverless Operator from 1.12 branch

### PR DESCRIPTION
**Pull request by navidshaikh**
/assign @dsimansk

### CODE REVIEW

File: openshift/e2e-tests-openshift.sh [hunk 1]
```diff
@@ -13,10 +13,15 @@ failed=0
 # Temp disabled due to running into OOM
 # (( !failed )) && run_unit_tests || failed=1
```
Comment (dsimansk) [inline openshift/e2e-tests-openshift.sh:14]: Can you enable unit tests also? It seems I've missed it in my PR to master. Those shouldn't be problematic anymore.

Comment (navidshaikh) [reply openshift/e2e-tests-openshift.sh]: enabled

**Pull request merged**